languages/clojure/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,45 @@
+In Clojure, binding a value to a name is referred to as a _var_. Top-level (global) vars are similar to constants in other languages, but are commonly redefined at a REPL to facilitate dynamic development.
```

Comment:
Not all students will know what a REPL is I think. Could you get away with doing:

```suggestion
In Clojure, binding a value to a name is referred to as a _var_. Top-level (global) vars are similar to constants in other languages, but are commonly redefined at to facilitate dynamic development.
```

Or does that not make sense?

Reply:
Sure, that works. Good point.